solrCloud_external_zooKeeper/init_files/security.json

```diff
@@ -0,0 +1,23 @@
+{
+    "authentication": {
+            "blockUnknown": true,
+            "class": "solr.BasicAuthPlugin",
+            "credentials": {
+            "solr": "IV0EHq1OnNrj6gvRCwvFwTrZ1+z1oBbnQdiVC3otuq0= Ndd7LKvVBAaZIF0QAVi1ekCfAJXr1GGfLtRUXhgrF8c="
```

Comment:
It may be worth a note in the README (can't put a comment in the json directly) indicating that this example `security.json` has the default `SolrRocks` password, just so we don't think later that we committed something secret here.